backend/service/k8s/configmap_test.go

```diff
@@ -90,6 +103,53 @@ func TestListConfigMaps(t *testing.T) {
 	assert.Len(t, result, 3)
 }
 
+func TestDescribeConfigMap(t *testing.T) {
+	t.Parallel()
+
+	cs := testConfigMapClientset()
+
+	s := &svc{
+		manager: &managerImpl{
+			clientsets: map[string]*ctxClientsetImpl{"testing-clientset": &ctxClientsetImpl{
+				Interface: cs,
+				namespace: "testing-namespace",
+				cluster:   "testing-cluster",
+			}},
+		},
+	}
+
+	configMap, err := s.DescribeConfigMap(context.Background(), "testing-clientset", "testing-cluster", "testing-namespace", "testing-configmap-name")
```

Comment:
Could you also add another case to Describe a configmap that does not exist and ensure there is an error

Reply:
added in https://github.com/lyft/clutch/pull/930/commits/147f0d0b6cd71926d777622693c9b8e90e5b7349

TIL that the fakeclient doesn't filter by field selectors. was raised in https://github.com/kubernetes/client-go/issues/326 which linked me to other issues. so I followed the pattern I've seen in other tests for "not found"  https://github.com/lyft/clutch/blob/main/backend/service/k8s/pods_test.go#L129